projects/packages/blaze/src/class-dashboard-config-data.php

```diff
@@ -87,6 +113,15 @@ public function get_data() {
 				),
 			),
 		);
+
+		/**
+		 * Filter to allow modification of the Blaze dashboard config data.
+		 *
+		 * @param bool $data Blaze dashboard config data.
+		 *
+		 * @since 0.20.3
```

Comment:
```suggestion
		 * @since $$next-version$$
```

If you do it that way, it will be automatically updated on release day, and this way you do not have to know what version to mention.

Reply:
Thanks! I will make that change.